### PR TITLE
Normalize exported Filepaths

### LINKED ISSUE #89: Mac/Win paths
I get borked levels when my mac opens up files that were saved using windows `foo\\bar` paths
Screen Shot 2019-12-02 at 11 10 30 PM

### PR DESCRIPTION
**Pull request by AustinEast**
closes #89 

This normalizes all exported filepaths (that i know of) to `"look/like/this"`. This'll also "auto-upgrade" projects that have different filepath styles `"like\\this"`, so no worries about breaking changes :)

### CODE REVIEW

File: src/project/data/Project.hx [hunk 1]
```diff
@@ -40,7 +40,7 @@ class Project
 	public function new(path:String)
 	{
 		this.name = "New Project";
-		this.path = Path.resolve(path);
+		this.path = Path.resolve(Path.normalize(path));
```
Comment (TheSpydog) [inline src/project/data/Project.hx]: Is this supposed to be `haxe.io.Path`? I don't see an import at the top

**Comment by TheSpydog** [comment]
What's the package-lock update about? Is that needed for this feature, or did that accidentally get caught up in the changes?

Also, I wonder if it might be better to just consistently use `haxe.io.Path` inline for every use here, just so there's no confusion which Path is being referred to when you're looking at a single line. (Eliminates the need to scroll up to the top and check the imports.)

File: src/project/data/Project.hx (hunk 1 above)
Comment (AustinEast) [reply src/project/data/Project.hx]: Nope, thats actually some leftover code i should delete. Its not really doing anything, but why risk it?

**Comment by AustinEast** [comment]
The package lock updates have been kind of "needed" (as in not really needed, but should be updated), since our package structure may have changed in some form from any of the dependencies. i dont think we need to worry about it too much at this point.

Also, good point about the inlining, ill do that.

**Pull request merged**